packages/component-library/package.json

```diff
@@ -397,9 +82,75 @@
+    "./Header/theme-switch": "./src/Header/theme-switch.tsx",
+    "./InfoBox": "./src/InfoBox/index.tsx",
+    "./Input": "./src/Input/index.tsx",
+    "./Link": "./src/Link/index.tsx",
+    "./MainNavTabs": "./src/MainNavTabs/index.tsx",
+    "./Meter": "./src/Meter/index.tsx",
+    "./MobileNavTabs": "./src/MobileNavTabs/index.tsx",
+    "./ModalDialog": "./src/ModalDialog/index.tsx",
+    "./NoResults": "./src/NoResults/index.tsx",
+    "./NotFoundPage": "./src/NotFoundPage/index.tsx",
+    "./Paginator": "./src/Paginator/index.tsx",
+    "./SearchButton": "./src/SearchButton/index.tsx",
+    "./SearchInput": "./src/SearchInput/index.tsx",
+    "./Select": "./src/Select/index.tsx",
+    "./SingleToggleGroup": "./src/SingleToggleGroup/index.tsx",
+    "./Skeleton": "./src/Skeleton/index.tsx",
+    "./Spinner": "./src/Spinner/index.tsx",
+    "./StatCard": "./src/StatCard/index.tsx",
+    "./Status": "./src/Status/index.tsx",
+    "./Switch": "./src/Switch/index.tsx",
+    "./SymbolPairTag": "./src/SymbolPairTag/index.tsx",
+    "./TabList": "./src/TabList/index.tsx",
+    "./Table": "./src/Table/index.tsx",
+    "./TableGrid": "./src/TableGrid/index.tsx",
+    "./TableGrid/table-grid-props": "./src/TableGrid/table-grid-props.ts",
+    "./Term": "./src/Term/index.tsx",
+    "./Virtualizer": "./src/Virtualizer/index.tsx",
+    "./compose-providers": "./src/compose-providers.tsx",
+    "./omit-keys": "./src/omit-keys.ts",
+    "./overlay-visible-context": "./src/overlay-visible-context.tsx",
+    "./package.json": "./package.json",
+    "./social-links": "./src/social-links.tsx",
+    "./theme": "./src/theme.scss",
+    "./unstyled/Breadcrumbs": "./src/unstyled/Breadcrumbs/index.tsx",
+    "./unstyled/Button": "./src/unstyled/Button/index.tsx",
+    "./unstyled/Dialog": "./src/unstyled/Dialog/index.tsx",
+    "./unstyled/GridList": "./src/unstyled/GridList/index.tsx",
+    "./unstyled/Label": "./src/unstyled/Label/index.tsx",
+    "./unstyled/Link": "./src/unstyled/Link/index.tsx",
+    "./unstyled/ListBox": "./src/unstyled/ListBox/index.tsx",
+    "./unstyled/Meter": "./src/unstyled/Meter/index.tsx",
+    "./unstyled/Popover": "./src/unstyled/Popover/index.tsx",
+    "./unstyled/SearchField": "./src/unstyled/SearchField/index.tsx",
+    "./unstyled/Select": "./src/unstyled/Select/index.tsx",
+    "./unstyled/Switch": "./src/unstyled/Switch/index.tsx",
+    "./unstyled/Table": "./src/unstyled/Table/index.tsx",
+    "./unstyled/Tabs": "./src/unstyled/Tabs/index.tsx",
+    "./unstyled/TextField": "./src/unstyled/TextField/index.tsx",
+    "./unstyled/Toolbar": "./src/unstyled/Toolbar/index.tsx",
+    "./use-prefetch": "./src/use-prefetch.ts",
+    "./useAlert": "./src/useAlert/index.tsx",
+    "./useCopy": "./src/useCopy/index.ts",
+    "./useData": "./src/useData/index.ts",
+    "./useDrawer": "./src/useDrawer/index.tsx",
+    "./useLogger": "./src/useLogger/index.tsx",
+    "./useQueryParamsPagination": "./src/useQueryParamsPagination/index.ts"
```

Comment:
can we use a wildcard so we don't have to maintain all the subpath exports manually?